ipaserver/install/server/upgrade.py

```diff
@@ -24,6 +24,7 @@
 import SSSDConfig
 import ipalib.util
 import ipalib.errors
+from ipalib import constants as ipalib_constants
```

Comment:
Please use `from ipalib.constants import IPAAPI_USER` to be consistent with the rest of the code.

Reply:
ack